core/arch/arm/mm/tee_pager.c

```diff
@@ -104,6 +104,48 @@ static struct tee_pager_pmem_head tee_pager_rw_pmem_head =
 /* Number of registered physical pages, used hiding pages. */
 static size_t tee_pager_npages;
 
+#ifdef CFG_TEE_CORE_EMBED_INTERNAL_TESTS
+struct tee_pager_stats _tee_pager_stats;
```

Comment:
`tee_pager_stats` would be good enough for me.

When we read/write statistics like this there's a chance (especially when we've gone multi core) that we get inconsistent values from time to time. Is that something we care about?